ramls/scheduled-notice.json

```diff
@@ -22,6 +22,11 @@
       "format": "date-time",
       "description": "Next run time"
     },
+    "triggeringEvent": {
```

Comment:
How i this intended to be used?

Why is this discriminator needed for expiration notices and not for other kinds of notices?

Reply:
It is intented for scheduled notices selection during request time based notices processing implemented in scope of this story https://issues.folio.org/browse/CIRC-387.

We also have due date event for timebased notices, currently it is not presented in the schema and is not used by due date scheduled notices processing job. It means that during due date scheduled notices processing all scheduled notices(request and due date) are retrieved by the job and the filtered by `loanId` property presence. The due date processing job is going to be reworked in scope of this story https://issues.folio.org/browse/CIRC-408 to use new due date triggering event. Due date triggering event will be added to the schema in scope of this story https://issues.folio.org/browse/CIRCSTORE-140.

Reply:
What about reminder notices? Is it a "due date" event or something else?

Reply:
What do you mean by reminder notices? Currently the only one triggering event for loan related time-based notices is `due date` (it may have different notice configurations, e.g. upon due date, before due date, after due date.

Reply:
As described in scenario 1's of various request types in [CIRC-387](https://issues.folio.org/browse/CIRC-387), there's a reminder notice to be sent. How is this notice type is mapped to the triggering events?

Reply:
From what I understand in first scenario reminder notice is not a specific triggering event. For example "it's nearing hold expiration" means that triggering event is `Hold Expiration` with before notice config.

Reply:
Can you explain how "before notice config" works? I am not familiar with events and schedulings. If you can just provide a link to this configuration that would be great as well.

Reply:
@mtraneis here is the link https://github.com/folio-org/mod-circulation-storage/blob/master/ramls/scheduled-notice.json#L25
Notice configuration is set based on notice policy when scheduled notice is created. For example when request with expiration date is created and there is notice policy configured for request expiration triggering event.